--- specifications/chef-server-0.8.16.gemspec.orig	2010-05-24 22:50:06.835857505 -0700
+++ specifications/chef-server-0.8.16.gemspec	2010-05-24 22:50:51.267753803 -0700
@@ -29,7 +29,6 @@
       s.add_runtime_dependency(%q<json>, ["<= 1.4.2"])
       s.add_runtime_dependency(%q<thin>, [">= 0"])
       s.add_runtime_dependency(%q<haml>, [">= 0"])
-      s.add_runtime_dependency(%q<ruby-openid>, [">= 0"])
       s.add_runtime_dependency(%q<coderay>, [">= 0"])
     else
       s.add_dependency(%q<merb-core>, ["~> 1.0.0"])
@@ -39,7 +38,6 @@
       s.add_dependency(%q<json>, ["<= 1.4.2"])
       s.add_dependency(%q<thin>, [">= 0"])
       s.add_dependency(%q<haml>, [">= 0"])
-      s.add_dependency(%q<ruby-openid>, [">= 0"])
       s.add_dependency(%q<coderay>, [">= 0"])
     end
   else
@@ -50,7 +48,6 @@
     s.add_dependency(%q<json>, ["<= 1.4.2"])
     s.add_dependency(%q<thin>, [">= 0"])
     s.add_dependency(%q<haml>, [">= 0"])
-    s.add_dependency(%q<ruby-openid>, [">= 0"])
     s.add_dependency(%q<coderay>, [">= 0"])
   end
 end
