--- specifications/chef-server-webui-0.8.16.gemspec.orig	2010-05-24 22:51:48.951762395 -0700
+++ specifications/chef-server-webui-0.8.16.gemspec	2010-05-24 22:52:08.944166988 -0700
@@ -30,7 +30,6 @@
       s.add_runtime_dependency(%q<json>, ["<= 1.4.2"])
       s.add_runtime_dependency(%q<thin>, [">= 0"])
       s.add_runtime_dependency(%q<haml>, [">= 0"])
-      s.add_runtime_dependency(%q<ruby-openid>, [">= 0"])
       s.add_runtime_dependency(%q<coderay>, [">= 0"])
     else
       s.add_dependency(%q<merb-core>, ["~> 1.0.0"])
@@ -42,7 +41,6 @@
       s.add_dependency(%q<json>, ["<= 1.4.2"])
       s.add_dependency(%q<thin>, [">= 0"])
       s.add_dependency(%q<haml>, [">= 0"])
-      s.add_dependency(%q<ruby-openid>, [">= 0"])
       s.add_dependency(%q<coderay>, [">= 0"])
     end
   else
@@ -55,7 +53,6 @@
     s.add_dependency(%q<json>, ["<= 1.4.2"])
     s.add_dependency(%q<thin>, [">= 0"])
     s.add_dependency(%q<haml>, [">= 0"])
-    s.add_dependency(%q<ruby-openid>, [">= 0"])
     s.add_dependency(%q<coderay>, [">= 0"])
   end
 end
