--- specifications/chef-server-api-0.9.6.gemspec.orig	2010-07-14 23:17:27.808282102 -0700
+++ specifications/chef-server-api-0.9.6.gemspec	2010-07-14 23:19:10.712134624 -0700
@@ -23,13 +23,13 @@
     s.specification_version = 3
 
     if Gem::Version.new(Gem::RubyGemsVersion) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<merb-core>, ["~> 1.1.0"])
-      s.add_runtime_dependency(%q<merb-slices>, ["~> 1.1.0"])
-      s.add_runtime_dependency(%q<merb-assets>, ["~> 1.1.0"])
-      s.add_runtime_dependency(%q<merb-helpers>, ["~> 1.1.0"])
-      s.add_runtime_dependency(%q<merb-param-protection>, ["~> 1.1.0"])
+      s.add_runtime_dependency(%q<merb-core>, [">= 0"])
+      s.add_runtime_dependency(%q<merb-slices>, [">= 0"])
+      s.add_runtime_dependency(%q<merb-assets>, [">= 0"])
+      s.add_runtime_dependency(%q<merb-helpers>, [">= 0"])
+      s.add_runtime_dependency(%q<merb-param-protection>, [">= 0"])
       s.add_runtime_dependency(%q<json>, ["<= 1.4.2"])
-      s.add_runtime_dependency(%q<uuidtools>, ["~> 2.1.1"])
+      s.add_runtime_dependency(%q<uuidtools>, [">= 2.1.1"])
       s.add_runtime_dependency(%q<thin>, [">= 0"])
     else
       s.add_dependency(%q<merb-core>, ["~> 1.1.0"])
